### PR TITLE
Code Review of Q7

### PR DESCRIPTION
**Pull request by Harshwardhanwtagline**

### CODE REVIEW

File: Q7.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import re
+
+def valid_email(emails):
+    #I have make pattern variable, on bases of given condition using regex. 
+    pattern = r'\b[A-Za-z0-9._-]+@[A-Za-z0-9]+\.[A-Za-z]{2,3}\b'
+    #I have loop over the emails_list, use match and remove function to remove the unsupport mail form list.   
+    for i in emails[:]:
```
Comment (priteshtagline) [inline Q7.py:7]: Change name of `i` to email. Also no need of writing emails[:], you can directly loop through it `for email in emails`

File: Q7.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        if not bool(re.match(pattern, i)):
+            emails.remove(i)
+    return emails
+
+
+
```
Comment (priteshtagline) [inline Q7.py:13]: Remove unnecessary spaces.

File: Q7.py (hunk 1 above)
Comment (himanshu-suthar-tagline) [inline Q7.py]: Change name `i` to `email`. You can use for loop directly on emails `for email in emails` no need to write emails[:]

File: Q7.py (hunk 2 above)
Comment (himanshu-suthar-tagline) [inline Q7.py]: Remove extra spaces

**Pull request merged**